### PR TITLE
Distribute other format books among mdbook books

### PR DESCRIPTION
**Pull request by glyn**
Fixes https://github.com/lborb/book/issues/29

### CODE REVIEW

File: src/title-page.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ This book is a treasure-trove of Rust books in [mdbook](https://github.com/rust-
 * [Official books](./official.md) maintained at [rust-lang.org](https://www.rust-lang.org/)
 * [Unofficial books](./unofficial.md) maintained elsewhere
 * [Application books](./applications.md) on specific Rust applications
-* [Books in Other Formats](./other-formats.md)
 
 If you know of others, why not submit a [pull request](https://github.com/lborb/book#pull-requests)?
+
+Some books get an honourable mention even though they aren't actually in
```
Comment (kw-fn) [inline src/title-page.md:10]: Can we remove use of word 'honourable' so that it reads 'Some books are mentioned even though..'. As well here I believe it is Rust Lang convention, as far as compiler internals and official docs are concerned, to use American English spelling, or do you prefer UK variants?

**Review by kw-fn — COMMENTED** [review]
I think that italics are an elegant way to represent non-'mdbook' books. Great idea!

File: src/title-page.md (hunk 1 above)
Comment (glyn) [reply src/title-page.md:10]: Nice catch! I'm English, so prefer English spellings, but am happy to oblige. Let me rephrase and force push an update.

**Review by kw-fn — APPROVED** [review]
I like that there is a section for FFI. All great! Go ahead and merge when ready.
